en/docs/integrations/build-your-first-integration.md

```diff
@@ -1,59 +1,133 @@
+  - Connect to the COVID-19 data API to get COVID-19 statistics.
+  - Connect to the world bank data API to get population data.
+  - Send an email with the summary of statistics to a specified email address.
 
-## Step 1: Configure and start the integration
-Follow this procedure to configure and start the **GitHub New Issue to Google Sheets Row** integration:
+## Step 1: Create the integration
+Follow this procedure to create an integration from scratch:
 
 1. Sign in to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/).
-2. Go to the **Integrations** card and click **Get Started**. Now you are on the **Integrations** page, where you can start with a prebuilt integration or create your own.
-3. Go to the **GitHub New Issue to Google Sheets Row** card and click **Use This**.
-4. In the **Configurations** panel, proceed as follows:
-    1. Click **Connect to GitHub**.
-    2. On the **Authorize** page that opens, click **Authorize wso2** to allow WSO2 to access your GitHub account. 
-    3. On the **Confirm Access** page that opens, enter your GitHub password and click **Confirm password**. This redirects you to the Choreo Console to continue the configuration.
-    4. Select the **GitHub Repository URL** depending on the repository for which you want to track new issues.
-    5. Click **Connect to Google Sheets** and then click **Proceed**.
-    6. Select the Google account to use in the integration and click **Allow** to proceed with the selected account.
-    7. Select `Sample GitHub Integration` as the **Spreadsheet Name**. 
-    8. Enter `Sheet1` as the **Worksheet Name**.
-5. Click **Save**.
- 
-    ![Save configuration](../assets/img/integrations/configurations-panel.png){.cInlineImage-half}
-    
-    Now you have configured the integration and you are ready to start it.
-
-6. Click **Start**. 
-   The following logs appear to indicate that the integration is starting:
+2. Go to the **Integrations** card and click **Get Started**. This takes you to the **Integrations** page.
+3. Click **Create**. This takes you to the **Create Integration** page.
+4. Go to the **Create with Choreo** card, enter `USACovidStatus` as the integration name, and then click **Create**.
+5. In the **Select Trigger** window, click **Manual**. This creates an integration that you can manually trigger.
   
-    ![Starting the integration](../assets/img/integrations/integration-starting.png){.cInlineImage-half}
-    
-    The following message indicates that the integration has started and is running:
 
-    ![Integration started](../assets/img/integrations/integration-started.png){.cInlineImage-full}
+## Step 2: Get the COVID-19 data 
+Follow this procedure to connect to the COVID-19 API and retrieve data:
 
-Now the **GitHub New Issue to Google Sheets Row** integration is successfully configured and running.
+1. Click **API Calls** and then click **Covid19 API**.
```

Comment:
```suggestion
1. Click **API Calls** and then select **Covid19 API**.
```

Reply:
As we are selecting from a list here, not just click one single icon, can we use `select` for those cases?

Reply:
Yeah. would make sense because we are selecting one from a set of connectors.